backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5754900_sys_GL_Journal_Balances_Tab.sql

```diff
@@ -0,0 +1,555 @@
+-- Run mode: SWING_CLIENT
+
+-- Table: GL_Journal_Balances_v
+-- 2025-05-15T14:49:07.950Z
+INSERT INTO AD_Table (AccessLevel,ACTriggerLength,AD_Client_ID,AD_Org_ID,AD_Table_ID,CloningEnabled,CopyColumnsFromTable,Created,CreatedBy,DownlineCloningStrategy,EntityType,ImportTable,IsActive,IsAutocomplete,IsChangeLog,IsDeleteable,IsDLM,IsEnableRemoteCacheInvalidation,IsHighVolume,IsSecurityEnabled,IsView,LoadSeq,Name,PersonalDataCategory,ReplicationType,TableName,TooltipType,Updated,UpdatedBy,WEBUI_View_PageLength,WhenChildCloningStrategy) VALUES ('3',0,0,0,542479,'A','N',TO_TIMESTAMP('2025-05-15 14:49:07.605000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'A','D','N','Y','N','N','Y','N','N','N','N','N',0,'GL Journal Balances','NP','L','GL_Journal_Balances_v','DTI',TO_TIMESTAMP('2025-05-15 14:49:07.605000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,0,'A')
```

Comment:
is IsView set to Y?

Reply:
thanks for the tip. Done here : https://github.com/metasfresh/metasfresh/pull/20827/files#diff-014f2bbe21dc9cda7920d4d8b93615d304eeacf1def5cb1f56cdd82b0ac87c2fR635